core/src/test/scala/com/pingcap/tispark/convert/ToDateTimeSuite.scala

```diff
@@ -30,20 +30,20 @@ class ToDateTimeSuite extends BaseDataSourceTest("test_data_type_convert_to_date
 
     val readA: java.sql.Timestamp = java.sql.Timestamp.valueOf("2019-11-11 11:11:11")
     val readB: java.sql.Timestamp = java.sql.Timestamp.valueOf("1990-01-01 01:01:01.999")
-    val readC: java.sql.Timestamp = java.sql.Timestamp.valueOf("1995-05-01 21:21:21.6")
+    val readC: java.sql.Timestamp = java.sql.Timestamp.valueOf("1995-05-01 21:21:21.666666")
 
     readRow1 = Row(1, null, null, null)
     readRow2 = Row(2, readA, readB, readC)
   }
 
-  test("Test Convert from java.lang.Long to DATETIME") {
+  ignore("Test Convert from java.lang.Long to DATETIME") {
```

Comment:
why is this test ignored?

Reply:
1. do not sure what long means? maybe microseconds or milliseconds from 1970-01-01
2. should timezone be considered?

will support after investigate

Reply:
what's the behavior in tidb side?

Reply:
after investigate mysql do not support insert long type to DATETIME

Reply:
I believe we can delete this test or check if it is not supported.